luaui/Widgets/camera_move_mmb_click.lua

```diff
@@ -0,0 +1,45 @@
+function widget:GetInfo()
+  return {
+    name      = "Camera Move Middle Mouse (MMB) Click",
+    desc      = "Alternate view movement for the middle mouse button & overhead camera",
+    author    = "qknight <js@lastlog.de>",
+    date      = "July 2024",
+    license   = "GNU GPL, v2 or later",
+    layer     = 1,     --  after the normal widgets
```

Comment:
why does it have to be after the normal widgets? is this an abnormal widget?

Reply:
I don't have a good reason, I just copied the other implementation as a template.

Reply:
@sprunk i put it on layer 0 now